README.md

```diff
@@ -360,6 +360,7 @@ parser_config.zimbu = {
     files = {"src/parser.c"},
     -- optional entries:
     branch = "main", -- default branch in case of git repo if different from master
+    location = "", -- location of the tree-sitter parser in the (larger) repository
```

Comment:
I think the default value is `nil`. Empty string is truthy in Lua

Reply:
Happy to fix the `nil` issue, and make it a bit more clear that this is for "non-standard" locations.
However, I do not understand the comment about the required start using `tree-sitter-{lang}`. I current am using `location = "misc/tree-sitter-{lang}` as my location and that seems to work perfectly fine. Does it have to start with `tree-sitter-{lang}`, or maybe end with `tree-sitter-{lang}`?